src/test/java/seedu/guest/model/guest/GuestTest.java

```diff
@@ -71,4 +73,29 @@ public void equals() {
         editedAlice = new GuestBuilder(ALICE).withEmail(VALID_EMAIL_BOB).build();
         assertFalse(ALICE.equals(editedAlice));
     }
+
+    @Test
+    public void hashcode() {
```

Comment:
Should we add tests for each field (e.g., different date range and number of guests)?